parser.go

```diff
@@ -378,7 +378,7 @@ func (p *Parser) parseColumnDefinition() (_ *ColumnDefinition, err error) {
 		return &col, err
 	}
 
-	if _, tok, lit := p.peekScan(); tok == IDENT && isTypeName(lit) {
+	if tok := p.peek(); tok == IDENT {
```

Comment:
Do you think the logic was too strict previously?

Reply:
It was. That's what was causing types like `UUID` to be detected as new columns.